src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/dagligReise/DagligReiseVedtakController.kt

```diff
@@ -45,6 +52,26 @@ class DagligReiseVedtakController(
         lagreVedtak(behandlingId, vedtak)
     }
 
+    @PostMapping("{behandlingId}/avslag")
+    fun avslå(
+        @PathVariable behandlingId: BehandlingId,
+        @RequestBody vedtak: AvslagDagligReiseDto,
+    ) {
+        val fagsakId = behandlingService.hentBehandling(behandlingId).fagsakId
+        val stønadsType = fagsakService.hentFagsak(fagsakId).stønadstype
+
+        when (stønadsType) {
+            Stønadstype.DAGLIG_REISE_TSO, Stønadstype.DAGLIG_REISE_TSR ->
+                validerGyldigÅrsakAvslag.validerAvslagErGyldig(
+                    behandlingId,
+                    vedtak.årsakerAvslag,
+                    stønadsType,
+                )
+            else -> logger.info("ingen avlag for daglig reise ")
+        }
```

Comment:
Er det nødvendig å ha en when her? Kunne man bare kjørt: 
```suggestion
                validerGyldigÅrsakAvslag.validerAvslagErGyldig(
                    behandlingId,
                    vedtak.årsakerAvslag,
                    stønadsType,
                )
        }
```